### PR TITLE
Add python memory leak debugging faq

### PR DESCRIPTION
**Pull request by hexylena**
@mvdbeek this is based on your message in the admins channel, is this OK for you?

### CODE REVIEW

File: topics/admin/faqs/python-memory-leaks.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+```console
+memray flamegraph --leaks --temporal the_dump -o the_dump.html
+```
+
+would then produce a report that shows allocation made but not freed over time.
+
+It might also be useful to just check what the process is doing with py-spy dump.
+
+If you're debugging a Galaxy process, and a user is splitting a dataset into a million element collection that could use some memory, however we have some pretty stringent limits for this now and haven't had a problem in a while.
```
Comment (mvdbeek) [inline topics/admin/faqs/python-memory-leaks.md:27]:
```suggestion
```
I guess that's not very relevant and taken out of context

Comment (mvdbeek) [inline topics/admin/faqs/python-memory-leaks.md:27]:
```suggestion
You can follow web workers in gunicorn with

memray run --follow-fork -o the_dump gunicorn 'galaxy.webapps.galaxy.fast_factory:factory()' --timeout 600 --pythonpath lib -k galaxy.webapps.galaxy.workers.Worker -b localhost:8082 --config python:galaxy.web_stack.gunicorn_config -w 1 --preload

the traced app will run on port 8082, you can then for instance in an upstream nginx section direct a portion of the traffic to your profiled app.
```

I guess that's not very relevant and taken out of context, instead this is how to follow gunicorn workers.

**Review by mvdbeek — DISMISSED** [review]
Thanks, good idea to record it here.

File: topics/admin/faqs/python-memory-leaks.md (hunk 1 above)
Comment (hexylena) [reply topics/admin/faqs/python-memory-leaks.md:27]: Sure, that makes sense!

**Comment by hexylena** [comment]
If you could re-approve @mvdbeek we'll get this merged, thanks for the additional command, that'll be useful

**Review by mvdbeek — APPROVED** [review]

**Pull request merged**